Documentation/IntroductionPackage/Index.rst

```diff
@@ -51,7 +51,19 @@ To install the Introduction Package run the following command:
 
          ddev composer require typo3/cms-introduction
 
-This command will download and activate the extension.
+Since TYPO3 11.5 it is essential to setup the extension afterwards by running
```

Comment:
I would prefer the `..  versionadded:: 11.5` directive, so like

    ..  versionadded:: 11.5
        The extension can be set up afterwards by running the following command:

Reply:
We will do that in a follow up